tests/test_plugin_loader.py

```
@@ -3670,32 +4374,45 @@ def tearDown(self):
+        self, mock_collect, mock_run, mock_logger
     ):
-        """Test dependency installation when disabled."""
-        mock_check_enabled.return_value = False
-        _install_requirements_for_repo(self.repo_path, "test-plugin")
-        mock_logger.warning.assert_called_with(
-            "Auto-install of requirements for %s disabled by config; skipping.",
-            "test-plugin",
+        """Community dependency auto-install should emit the risk warning once."""
+        mock_collect.return_value = ["requests==2.28.0"]
+        mock_run.return_value = subprocess.CompletedProcess(args=[], returncode=0)
+        with patch.object(pl, "_community_dep_install_warning_logged", False):
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)